pkg/apis/core/types.go

```diff
@@ -3594,17 +3594,18 @@ const (
 	ServiceTypeExternalName ServiceType = "ExternalName"
 )
 
-// ServiceInternalTrafficPolicyType describes the type of traffic routing for
+// ServiceInternalTrafficPolicy describes the type of traffic routing for
 // internal traffic
-type ServiceInternalTrafficPolicyType string
+type ServiceInternalTrafficPolicy string
+type ServiceInternalTrafficPolicyType = ServiceInternalTrafficPolicy
```

Comment:
@thockin missing the // for backwards compat comment here? Or maybe we don't need it?